gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -94,8 +94,8 @@ class BaselineIdea extends AbstractBaselinePlugin {
             }
         }
 
-        def lastfileName = copyrightDir.relativize(copyrightFiles.iterator().toList().last().toPath())
-        copyrightManager.@default = lastfileName
+        def firstFileName = copyrightDir.relativize(copyrightFiles.iterator().toList().first().toPath())
+        copyrightManager.@default = firstFileName
```

Comment:
is this going to break existing setups? could just invert my 000_ and 001_ stuff.